modules/openapi-generator/src/main/resources/swift4/APIs.mustache

```diff
@@ -11,6 +11,7 @@ open class {{projectName}}API {
     public static var credential: URLCredential?
     public static var customHeaders: [String:String] = [:]
     public static var requestBuilderFactory: RequestBuilderFactory = AlamofireRequestBuilderFactory()
+    public static var queue: DispatchQueue = .main
```

Comment:
@syatyo what do you think of giving this a more specific name?
Something like `APIResponseQueue` or something like that.
It's just to avoid confusion in case more configurable queue's are added in the future.

Reply:
Variable names in swift start with lowercase, so maybe something like `apiResponseQueue`?

Reply:
It seems better! I will try it.